### PR TITLE
Changed debezium installer script

### PR DESCRIPTION
**Pull request by ShivanshGahlot**
Merged the install-yb-voyager-debezium script into install-yb-voyager-script
Made a new script for internal testing of debezium: `/migtests/scripts/build-and-install-yb-voyager-debezium`

The first script installs latest release of debezium and voyager by default. If we pass the option `--install-from-local-source`, it installs the latest commit from debezium voyager-main branch and builds and installs yb-voyager from local branch.

### CODE REVIEW

File: installer_scripts/install-yb-voyager-debezium [hunk 1]
```diff
@@ -4,15 +4,15 @@ export LC_CTYPE=en_US.UTF-8
...
 trap on_exit EXIT
 LOG_FILE=/tmp/install-yb-voyager-debezium.log
 YUM="sudo yum install -y -q"
 
 # Remember to update the S3 repo with the correct version of pre-biult debezium-server, if any change in these values.
-DEBEZIUM_VERSION="2.2.0-SNAPSHOT"
-if [ -z "$DEBEZIUM_REF" ]; then
-	DEBEZIUM_REF="8d2043e631eca4e527963b4a8f2f364a2c355f1b"
-fi
+DEBEZIUM_TARBALL_VERSION="latest"
```
[the remark this answers is not in the document]
Comment (makalaaneesh) [reply installer_scripts/install-yb-voyager-debezium:15]: We'll need it when we want to pin to a particular release version as well. (when installing via brew)

File: installer_scripts/install-yb-voyager-debezium [hunk 2]
```diff
@@ -71,6 +72,29 @@ main() {
...
+		OPTS=$(getopt -o "l", --long latest --name 'install-yb-voyager-debezium' -- $ARGS_LINUX)
+	else
+		OPTS=$(getopt  l  $ARGS_MACOS)
+	fi
+
+	eval set -- "$OPTS"
+
+	while true; do
+		case "$1" in
+			-l | --latest )
```
Comment (makalaaneesh) [inline installer_scripts/install-yb-voyager-debezium:87]: the default is set to latest, anyway. Do we need this option then?

File: installer_scripts/install-yb-voyager [hunk 3]
```diff
@@ -173,6 +183,36 @@ macos_main() {
...
+		JAVA_MAJOR_VER=$(${JAVA_BINARY} -version 2>&1 | awk -F '"' '/version/ {print $2}' | awk -F. '{print $1}')
+		
+        if ([ -n "$JAVA_MAJOR_VER" ] && (( 10#${JAVA_MAJOR_VER} >= 10#${MIN_REQUIRED_MAJOR_VERSION} )) ) #integer compare of versions.
+        then
+        	output "Found sufficient java version = ${JAVA_MAJOR_VER}"
+        else
+        	output "ERROR: Java not found or insuffiencient version ${JAVA_MAJOR_VER}. Please install java>=${MIN_REQUIRED_MAJOR_VERSION}"
+			exit 1;
+        fi
+}
```
Comment (ShivanshGahlot) [inline installer_scripts/install-yb-voyager:203]: @amit-yb do you still want the java version check to be simplified to just a binary present or not check?

File: installer_scripts/install-yb-voyager [hunk 4]
```diff
@@ -173,6 +183,36 @@ macos_main() {
...
+
+install_debezium_server() {
+	output "Installing debezium"
+	debezium_server_filename="debezium-server-${DEBEZIUM_TARBALL_VERSION}.tar.gz"
+	# download
+	wget -nv "https://github.com/yugabyte/debezium/releases/download/voyager-debezium/${debezium_server_filename}"
```
Comment (makalaaneesh) [inline installer_scripts/install-yb-voyager:209]: As discussed in https://yugabyte.slack.com/archives/C04MC0N7B1A/p1684320717861679?thread_ts=1684315471.715119&cid=C04MC0N7B1A, make this `wget -nv "https://github.com/yugabyte/debezium/releases/download/voyager-debezium-${DEBEZIUM_TARBALL_VERSION}/${debezium_server_filename}"`

**Review by amit-yb — COMMENTED** [review]
Couple of minor comments.

File: installer_scripts/install-yb-voyager (hunk 4 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager]: Done

File: installer_scripts/install-yb-voyager (hunk 3 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager:199]: Done

**Review by amit-yb — APPROVED** [review]
LGTM.

File: installer_scripts/install-yb-voyager [hunk 5]
```diff
@@ -10,6 +10,10 @@ ARGS_LINUX=$@
 
 VERSION="latest"
 
+# This needs to be changed to the latest debezium release tag
+DEBEZIUM_BASE_VERSION="2.2.0-1.3.0"
```
Comment (makalaaneesh) [inline installer_scripts/install-yb-voyager:14]: How about we call this `DEBEZIUM_RELEASE_TAG`?

**Review by makalaaneesh — APPROVED** [review]
LGTM with minor comment

**Review by amit-yb — APPROVED** [review]
LGTM. Just take care of the comment from Aneesh.

**Review by sanyamsinghal — APPROVED** [review]
LGTM

**Review by sanyamsinghal — APPROVED** [review]
LGTM

**Pull request merged**